.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,78 @@
+name: Bug Report
+description: Report a bug encountered while operating RumbleDB
+labels: bug
+body:
+  - type: textarea
+    id: expected
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+
+  - type: textarea
+    id: repro
+    attributes:
+      label: How can we reproduce it (as minimally and precisely as possible)?
+    validations:
+      required: true
+
+  - type: textarea
+    id: additional
+    attributes:
+      label: Anything else we need to know?
+
+  - type: textarea
+    id: RumbleVersion
+    attributes:
+      label: RumbleDB version
+      value: |
+        <details>
+        after running the RumbleDB, it will show version next to the logo.
+        </details>
+    validations:
+      required: true
+
+  - type: textarea
+    id: osVersion
+    attributes:
+      label: OS version
+      value: |
+        <details>
+
+        ```console
+        # On Linux:
+        $ cat /etc/os-release
+        # paste output here
+        $ uname -a
+        # paste output here
+
+        # On Windows:
+        C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
+        # paste output here
+        ```
+
+        </details>
+    validations:
+      required: true
```

```suggestion
      required: false
```
I still think we can make this optional, since its not related to many issues and makes issue creation easier. Its good to have for things like terminal behaviour, but not needed for things like spec edgecase not correctly handled (which is evaluated on gitlab cicd where we dont even know this info)